### PR TITLE
[v12] Backport of dependabot CVE updates

### PR DESCRIPTION
**Pull request by jentfoo**
Cherry-picked security updates that have been applied in master but failed to backport to v12

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -37,7 +37,7 @@ require (
 	github.com/coreos/go-oidc v2.1.0+incompatible // replaced
 	github.com/coreos/go-semver v0.3.0
 	github.com/creack/pty v1.1.18
-	github.com/crewjam/saml v0.4.12
+	github.com/crewjam/saml v0.4.13
 	github.com/datastax/go-cassandra-native-protocol v0.0.0-20220706104457-5e8aad05cf90
 	github.com/dustin/go-humanize v1.0.1
 	github.com/elastic/go-elasticsearch/v8 v8.6.0
```
Comment (codingllama) [inline go.mod:43]: Missing piv-go below?

**Review by codingllama — COMMENTED** [review]
Why draft?

File: go.mod (hunk 1 above)
Comment (codingllama) [reply go.mod:43]: It may be easier to manually the deps you care about. Cherry-picking go.mod changes is rarely worth the trouble.

**Comment by zmb3** [comment]
Hey @jentfoo just want to make sure everything is correct here. The PR title mentions v12, but the commit message says v11.

**Comment by jentfoo** [comment]
@zmb3 That is due to cherry-picking from the v11 merge.  V11 referenced v10 in the same way.  I can amend the comment if you prefer, I just left it as the default.

**Comment by zmb3** [comment]
IMO we should always strive to have an accurate commit message. I would amend it.

**Review by codingllama — APPROVED** [review]

**Review by rosstimothy — APPROVED** [review]

**Pull request merged**